### PR TITLE
feat: create SaleOrderLine list screen

### PR DESCRIPTION
**Pull request by vhu-axelor**
RM#80385

### CODE REVIEW

**Comment by vhu-axelor** [comment]
PR #605 need to be merged first

File: packages/apps/sales/src/components/atoms/SaleOrderLineCard/SaleOrderLineCard.tsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+          items: [
+            {
+              displayText: productName,
+              isTitle: true,
+            },
+            {
+              displayText: description,
+              numberOfLines: 2,
+              hideIfNull: true,
+            },
```
Comment (lme-axelor) [inline packages/apps/sales/src/components/atoms/SaleOrderLineCard/SaleOrderLineCard.tsx:156]: Description is HTML field

File: packages/apps/sales/src/components/atoms/SaleOrderLineCard/SaleOrderLineCard.tsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+          items: [
+            !isStartOfPack && {
+              customComponent: (
+                <TextUnit
+                  style={styles.sideText}
+                  value={formatPrice(total)}
+                  unit={currencySymbol}
+                  defaultColor
+                />
+              ),
```
Comment (lme-axelor) [inline packages/apps/sales/src/components/atoms/SaleOrderLineCard/SaleOrderLineCard.tsx:200]: Should be diisplay only when activated on end of pack
Also please add margin with card right border

File: packages/apps/sales/src/screens/saleOrder/SaleOrderLineListScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        translator={I18n.t}
+      />
+    </Screen>
+  );
+};
+
+const styles = StyleSheet.create({
+  scrollView: {
+    height: null,
+  },
```
Comment (lme-axelor) [inline packages/apps/sales/src/screens/saleOrder/SaleOrderLineListScreen.tsx:89]: remove this

**Pull request merged**